src/components/aeTextInputMolecule/aeTextInputMolecule.vue

```diff
@@ -0,0 +1,39 @@
+<template>
+  <div class="ae-text-input-molecule">
+    <div class="top-container">
+      <label class="label" v-if="!!label" :for="internalInputId">{{label}}</label>
+      <span
+        class="top-message _type_error"
+        v-if="!!_errorMessage"
+        ref="errorMessage"
+      >
+        {{_errorMessage}}
+      </span>
+      <span
+        class="top-message _type_valid"
+        ref="validMessage"
+        v-if="!!_validMessage"
+      >
+        {{_validMessage}}
+      </span>
+    </div>
+    <ae-validated-text-input
+      :inputId="internalInputId"
```

Comment:
`_uid` can be used instead of `internalInputId`, probably not very good to use internal vue api, but it simpler.

Reply:
Didn't know about that. I think it's actually better. Because Vue will ensure that the ids are unique. Right know it's just probable that they are unique.

Reply:
fixed